### ISSUE TITLE
Some final small comments on the paper

### ISSUE THREAD

**Issue by dfm** [fbartolic/volcano#20]
- Abstract: "fully Bayesian" remove "fully" or replace this with probabilistic.
- Throughout: see if you can move the figures so that they don't always appear right below a section heading. It becomes hard to follow!
- Bottom of page 3: "Because we have lots of prior information on..." -> "Because of high-resolution resolved imaging of Io, we..."
- Eq 10: Just put the actual Matern kernel here rather than the approximation
- Figure 7 and 8: I think the title of the second map should be "inferred map" or something right?
- Throughout: I don't love the word "superior". I'd prefer something like "better" or (even better!) specifically describe how it's "better". For example, "more rigorous", "more physically motivated", or "less restrictive"...
- Page 31: "the fact that we cannot account for" -> "the fact that we do not yet account for"
- Page 38: "lots of divergences" -> "many divergences"

**Comment by rodluger** [comment]
Final comments! I'm officially signing off now. Good job!

- Add an explicit sentence somewhere (either at the end of the Intro or the end of the discussions) linking to your Github repository (people may not realize the icon at the end of the abstract is a link!)
- Don't capitalize *Heat-pipe volcanism* on the first page
- Citation in second paragraph of intro should be `Hammond & Pierrehumbert 2017`, not `Hammond & T. Pierrehumbert 2017`
- Citation in first paragraph of section 2 should be `Rayner et al. 2016`, not `John Rayner et al. 2016`
- Section 7.2: `Besides being useful for modeling secondary eclipses of volcanic Super-Earths, our model is also likely to` **`be`** `useful for modeling features on gaseous exoplanets with sparse surface features.`

**Comment by fbartolic** [comment]
Thanks! I did all of that.
